packages/babel-parser/test/fixtures/typescript/static-blocks/static-blocks-and-static-property-named-static/input.ts

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  static foo;
```

Comment:
As I read the name of the test, did you mean
```suggestion
  static static;
```
?